recipes/Axelrod/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "Axelrod" %}
+{% set version = "2.10.0" %}
+{% set sha256 = "9c5ecf5e6b5a9f8931888ec8305de7814585d9a10c75ad18cde93a80b8e45fa6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  
+  run:
+    - python
+    - pyparsing ==2.1.1
+    - numpy >=1.9.2
+    - matplotlib >=1.4.2
+    - hypothesis >=3.2
+    - tqdm ==3.4.0
```

Comment:
Here are [the changes between 3.4.0 and 3.8.0](https://github.com/tqdm/tqdm/compare/8e56299...0328832) ... I have no idea why that's a hard requirement. I seems likely it would work with 3.8.0 (last one before 4.0, which might signal more significant changes)